recipes/autots/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Since this is included in the PyPI tarball, you don't need to (and shouldn't) include it here.

Reply:
Thanks @dopplershift  !
There's so much talk of License file in the documentation that I had thought it best to include

Reply:
Yeah, understood. It's a tad confusing.